README.md

```diff
@@ -24,11 +32,21 @@ git clone https://github.com/Altinn/app-frontend-react
 cd app-frontend-react
 ```
 
+The development server can be started by following these steps:
+
+- Navigate to `./src/altinn-app-frontend`
+- `yarn --immutable` (only needed when `package.json` has changed)
+- `yarn start` (to start the development server)
+
 ### Developing app-frontend
 
-You need an Altinn app to effectively make changes to the app-frontend codebase. To serve the development version of app-frontend code, you need to make some changes to `views/Home/Index.cshtml` in the app repo you are using:
+You need an Altinn app to effectively make changes to the app-frontend codebase.
 
-Change
+Localtest now includes a way to avoid editing your app to use different frontends
```

Comment:
Instead of adding an image to the repo and describe this separately here, we could add a link to the docs instead: https://docs.altinn.studio/app/testing/local/debug/#using-other-frontend-versions

Reply:
👍

Reply:
Finally had some time in front of the computer.